OpenRA.Game/Game.cs

```diff
@@ -457,6 +465,28 @@ static string ChooseShellmap()
 			return shellmaps.Random(CosmeticRandom);
 		}
 
+		public static void SwitchToExternalMod(ExternalMod mod, string[] launchArguments = null, Action onFailed = null)
+		{
+			try
+			{
+				var argsString = mod.LaunchArgs.Append(launchArguments)
+					.Select(a => "\"" + a + "\"").JoinWith(" ");
+
+				var p = Process.Start(mod.LaunchPath, argsString);
```

Comment:
To clarify: in IRC I was told that this is not supposed to work in conjunction with https://github.com/OpenRA/OpenRA/issues/8993 especially not https://github.com/OpenRA/OpenRA/issues/9028 and https://github.com/OpenRA/OpenRA/issues/9029. So while this is still rickety in terms of launching some kind of program in the hopes that the ABI will swallow the parameters and do the right thing, this will probably not be used to distribute malicious software hidden in .oramap ZIPs that are installed via a mod switch.